exatomic/base.py

```diff
@@ -23,12 +24,23 @@
 sym2mass = {}
 sym2radius = {}
 sym2color = {}
+dfs = []
 for k, v in vars(isotopes).items():
     if isinstance(v, isotopes.Element):
         sym2mass[k] = v.mass
-#        sym2radius[k] = v.radius
         sym2radius[k] = [v.cov_radius, v.van_radius]
         sym2color[k] = '#' + v.color[-2:] + v.color[3:5] + v.color[1:3]
+        abds = list(map(lambda x: x.af, v.isotopes))
+        masses = list(map(lambda x: x.mass, v.isotopes))
+        isos = list(map(lambda x: x.A, v.isotopes))
+        z = list(map(lambda x: x.Z, v.isotopes))
+        symbol = list(map(lambda x: x.symbol, v.isotopes))
```

Comment:
Essentially this is a loop over `v.isotopes`: maybe it would be more efficient to explicitly loop rather than do 5 loops?

Reply:
Like maybe there is a more efficient way to do this whole looping?

Reply:
Removed in the latest commit